modules/misc/doc/content/source/auxkernels/CoupledDirectionalMeshHeightInterpolation.md

```diff
@@ -1 +1,15 @@
-!template load file=stubs/moose_object.md.template name=CoupledDirectionalMeshHeightInterpolation syntax=/AuxKernels/CoupledDirectionalMeshHeightInterpolation
\ No newline at end of file
+# CoupledDirectionalMeshHeightInterpolation
+
+!syntax description /AuxKernels/CoupledDirectionalMeshHeightInterpolation
+
+## Description
+
+This object scales a user-provied coupled variable by its position relative to the maximum and minimum coordinate of the model in the user-specified direction. When the coordinate is equal to the minimum coordinate, the scaling factor is 0, while when it is at the maximum coordinate, the scaling factor is 1, the scaling factor is linearly interpolated between those bounding values.
```

Comment:
`auxvariable` instead of `variable`? Not sure if other documentation differentiates or not.

Reply:
This would work on an AuxVariable or a Variable, which is why I intentionally didn't say "AuxVariable".  Maybe I should just say the it would work on either. I will note that this only applies to the undisplaced mesh.

Reply:
Oh I see, you're not operating on the variable, you're taking the value and passing it to a new auxvariable...of course.